packages/react-ui-core/src/Carousel/PhotoCarousel.js

```diff
@@ -0,0 +1,98 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+import themed from 'react-themed'
+import classnames from 'classnames'
+import LazyLoad from 'react-lazyload'
+import autobind from 'autobind-decorator'
+import Carousel from './Carousel'
+
+@themed(['PhotoCarousel'])
+
+export default class PhotoCarousel extends PureComponent {
+  static propTypes = {
+    className: PropTypes.string,
+    theme: PropTypes.object,
+    items: PropTypes.arrayOf(
+      PropTypes.shape({
+        caption: PropTypes.string,
+        path: PropTypes.string,
+      }),
+    ),
+    sizes: PropTypes.oneOfType([
+      PropTypes.array,
+      PropTypes.string,
+    ]),
+    server: PropTypes.string.isRequired,
+    lazyLoad: PropTypes.oneOfType([
+      PropTypes.bool,
+      PropTypes.object,
+    ]),
+  }
+
+  static defaultProps = {
+    lazyLoad: false,
+    theme: {},
+  }
+
+  photo(path) {
+    const { server, sizes } = this.props
+    return `${server}${path}${sizes}`
+  }
+
+  lazyLoad() {
+    return (
+      <LazyLoad>
```

Comment:
I see you set up lazyLoad as either a bool or an object, but you never seem to use lazyLoad as an object anywhere. Oversight?

Reply:
I meant to add it in as props here, need to check if it's an object here first.